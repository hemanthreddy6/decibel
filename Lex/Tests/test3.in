loop over i 10 to 20 {
    v <- 10;
}
