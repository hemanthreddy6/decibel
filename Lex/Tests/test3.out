LOOP
OVER
IDENTIFIER
INT_LITERAL
TO
INT_LITERAL
{
IDENTIFIER
<-
INT_LITERAL
;
}
